actionpack/test/controller/resources_test.rb

```diff
@@ -1115,12 +1115,11 @@ def with_restful_routing(*args)
       collection_methods = options.delete(:collection)
       member_methods = options.delete(:member)
       path_prefix = options.delete(:path_prefix)
-      args.push(options)
 
       with_routing do |set|
         set.draw do
           scope(path_prefix || "") do
-            resources(*args) do
+            resources(*args, **options) do
```

Comment:
Makes sense to update the signature of `with_restful_routing` to use kwargs too.

Reply:
This is a test method, so it doesn't matter a much, but I did it anyway.